src/main/java/frc/robot/subsystems/DriveBaseSubsystem.java

```diff
@@ -227,6 +230,9 @@ public void periodic() {
     // update odometryreset
     double leftPosition = leftEncoder.getPosition();
     double rightPosition = rightEncoder.getPosition();
+
+    sbGyroInfo.setDouble(gyro.getAngle());
```

Comment:
We should remove this, with the dial, this is another network call that isn't necessary